integration_tests/test_users_real_api.py

```diff
@@ -0,0 +1,139 @@
+import json
+import os
+
+from jcapiv1 import ApiClient, Configuration, Systemuserslist, Systemuserputpost
+
+from jccli import cli
+from click.testing import CliRunner, Result
+from jcapiv1.api.systemusers_api import SystemusersApi
+
+
+
+class TestUsersRealApi:
+    @classmethod
+    def setup_class(cls):
+        configuration = Configuration()
+        api_key = os.getenv('JC_API_KEY')
+        configuration.api_key['x-api-key'] = api_key
+        cls.systemusers_api = SystemusersApi(ApiClient(configuration=configuration))
+        cls.api_key = api_key
+
+        current_users: Systemuserslist = cls.systemusers_api.systemusers_list(
+            content_type='application/json',
+            accept='application/json'
+        )
+        assert current_users.total_count == 0
+
+    def test_create_user(self):
+        USERNAME = 'fakeuser567'
+        EMAIL = 'fakeemail3@fakesite.org'
+
+        runner = CliRunner()
+
+        result = runner.invoke(cli.cli, [
+            '--key',
+            self.api_key,
+            'user',
+            'create',
+            '--username',
+            USERNAME,
+            '--email',
+            EMAIL
+        ])
+        if result.exit_code:
+            raise ValueError(
+                "create-user exited with status code: %s;\nmessage was: %s" % (result.exit_code, result.exception)
+            )
+
+        generated_id = json.loads(result.output)['id']
+
+        # Use API to delete created user
+        self.systemusers_api.systemusers_delete(
+            accept='application/json',
+            content_type='application/json',
+            id=generated_id
+        )
+
+    def test_delete_user(self):
+        USERNAME = 'fakeuser789'
+        EMAIL = 'fakeemail4@fakesite.org'
+
+        # Create user to be deleted:
+        self.systemusers_api.systemusers_post(
+            content_type='application/json',
+            accept='application/json',
+            body=Systemuserputpost(
+                username=USERNAME,
+                email=EMAIL
+            )
+        )
+
+        runner = CliRunner()
+
+        result = runner.invoke(cli.cli, [
+            '--key',
+            self.api_key,
+            'user',
+            'delete',
+            '--username',
+            USERNAME
+        ])
+        if result.exit_code:
+            raise ValueError(
+                "get-user exited with status code: %s;\nmessage was: %s" % (result.exit_code, result.exception))
+        # TODO: what output should we expect from delete-user
```

Comment:
i would expect a 200 response and no text output on success.

Reply:
Makes sense, but how does the CLI runner test that there was a 200 response on the API triggered by the given CLI command?

Reply:
well that was metaphoric. running with cli delete user command should return exit code 0 and no text output if it did it's job.